static/js/analytics.js

```diff
@@ -312,11 +313,26 @@ const renderHashtagsGraphFromJson = (json) => {
   });
 };
 
+const renderGeotaggetGraphFromJsom = (json) => {
+  const geotaggedCoords = json
```

Comment:
Above you have const NGPSLocations 

Edit it there to have GPSLocations = json.filter(...);

Then you check the lenght at line 130 and then you use the .map((location) on top of it here, so you don't have the "location" check twice